src/mapml-viewer.js

```diff
@@ -525,6 +547,39 @@ export class MapViewer extends HTMLElement {
               {target: this}}));
       }
     });
+    var locateControl = this._geolocationButton._container;
+    var observer = new MutationObserver(function(mutations) {
+      // Check the current state of the control
+      if (locateControl.classList.contains('active') && locateControl.classList.contains('following')) {
+        locateControl.firstChild.title = M.options.locale.btnLocTrackOn;
+      } else if (locateControl.classList.contains('active')) {
+        locateControl.firstChild.title = M.options.locale.btnLocTrackLastKnown;
+      } else {
+        locateControl.firstChild.title = M.options.locale.btnLocTrackOff;
+      }
+    });
+    // Configure the observer to watch for changes to the class name
+    var observerConfig = { attributes: true, attributeFilter: ['class'] };
+    observer.observe(locateControl, observerConfig);
+
+    this._map.on('locationfound',
+      function (e) {
+        this.dispatchEvent(new CustomEvent('maplocationfound', {detail:
+          {latlng: e.latlng,     radius: e.accuracy}
```

Comment:
any specific reason for naming this property `radius`? I think `accuracy` might be more useful! Will need to somehow document this later that this number is in meters.